README.md

```diff
@@ -269,6 +269,82 @@ mv s3://bucket/2020/03/18/file1.gz s3://bucket/2020/03/18/original/file.gz
 ls # inline comments are OK too
 ```
 
+#### Sync
+With `sync` command, it's possible to synchronise files between an S3 bucket and a local directory;
```

Comment:
I read that as we only support s3->dir sync. Let's elaborate, like "`sync` command synchronizes S3 buckets, prefixes and directories. It also can synchronize given S3 buckets or prefixes.".

Reply:
Updated the sentence